code/game/objects/items/devices/oxycandle.dm

```diff
@@ -0,0 +1,72 @@
+/obj/item/device/oxycandle
+	name = "Oxygen Candle"
+	desc = "Oxygen candles are flare-like steel tubes of sodium chlorate that produces oxygen through chemical process."
```

Comment:
Grammar here is weird, and the phrasing feels out of place too. This should be a casually-worded description, not a dictionary definition.

Reply:
I am sorry, I am not a native speaker. And this was a place-holder during development.